### DISCUSSION TITLE
Welcome to localized_rich_text Discussions!

### DISCUSSION THREAD

**Discussion by LaTrita97** [LaTrita97/localized_rich_text#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AtamyratBabayev** [comment]
Hi @LaTrita97!
Thank you for your great job!

Are there any plans to add [GestureRecognizer] on [LocalizedRichTextKey] parts?
If no, I could contribute, because I really need this feature in my current project.

**Reply by LaTrita97** [reply]
Hi @Mickey-A-Mouse, thank you for your appreciation!
By the end of the week, I'll try to add this feature.

**Reply by AtamyratBabayev** [reply]
I've already created a pull request, waiting for your review.

**Reply by AtamyratBabayev** [reply]
> Hi @Mickey-A-Mouse, thank you for your appreciation! By the end of the week, I'll try to add this feature.

Wow, it was fast)
When will be updated pub.dev package?

**Reply by LaTrita97** [reply]
It's already online! @Mickey-A-Mouse

**Reply by LaTrita97** [reply]
I made a slight change by adding all the properties of the `TextSpan`, in case someone would also need other properties apart from the `GestureRecognizer`.
